docs/site_navigation/redirects.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Redirects
+parent: Site Navigation
+nav_order: 3
+---
+
+# Redirects
+
+The URL redirects are managed in **My Site > Redirects**. For and up-to-date full reference please consult [Set up 301 redirects for your website](https://support.easol.com/hc/en-gb/articles/4404999175953-Set-up-301-redirects-for-your-website) at the [Easol Help Centre](https://support.easol.com/hc/en-gb).
```

Comment:
For 'an' up-to-date full reference. 
But I thought we discussed the fact that the help documents are not necessarily kept up to date and we should have documentation for our application, particularly around how it works with redirects to unpublished pages and with series. Fair that you will need Eng's input to know exactly how this is working currently. @murraycatto is a good person to speak to as he made the latest updates.

Reply:
I'm updating according to [path_blocklist.rb](https://github.com/easolhq/easol/blob/3042bbfa5be98b3891af1559f648b645360d2be5/lib/site/path_blocklist.rb)